### ISSUE TITLE
convert function for jupyter notebook

### ISSUE THREAD

**Issue by rettigl** [Arora0/nomad-parser-nexus#13]
Make a convert() function that exports the interface of the convert.py file to be used within a jupyter notebook. Instead of passing the intermediate h5 file, directly pass the xarray object.

**Comment by Arora0** [comment]
So the role of the function would be to take in all the input files that the convert.py command takes (but doesn't use the command line interface) with a change that instead of the h5 file it takes in the xarray. This would require a small change in the reader. Would you still have the option of having the h5 file as an input since the converter will not run from the command line otherwise anymore, from what I understand.

**Comment by rettigl** [comment]
yes, off course this option should also continue to exist. Also, passing the h5 file from the jupyter notebook should be possible.
